miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -9,28 +9,41 @@ HotTarget.pool = {
   createColumns : function(config, create, data) {
     return [
         {
+          header : 'Pool Name',
+          data : 'name',
+          readOnly : true,
+          include : true,
+          unpackAfterSave : true,
+          unpack : function(pool, flat, setCellMeta) {
+            flat.name = pool.name;
+          },
+          pack : function(pool, flat, errorHandler) {
+            pool.name = flat.name;
```

Comment:
The confusion of fields mysteriously vanishing at a particular step is exactly my point. This is the only point at which the name disappears for some entities. It doesn't have any actual effect though, so as your preference I will match the existing pattern